### ISSUE TITLE
Mutation missing from output

### ISSUE THREAD

**Issue by mattdmem** [virus-evolution/gofasta#28]
Hello,

<img width="1136" alt="Screenshot 2022-01-17 at 10 02 39" src="https://user-images.githubusercontent.com/16154750/149748950-a7f749f7-320d-4386-8941-c6cc6a2e5d6a.png">

This snp in B.1.640.1 seems to be missing from the mutations list/frequency table (It's also missing from the B.1.640.2 mutation report). Any Ideas? 

Last few lines:
```
aa:ORF10:V32L(nuc:G29651T),0.004618938
del:29731:21,0.002309469
del:29737:18,0.004618938
del:29738:3,0.002309469
del:29738:21,0.581986143
del:29751:8,0.002309469
```

So the deletion is there, then no snp - is it due to sequence length?

Thanks

Matt

**Comment by mattdmem** [comment]
I tired just taking one of the sequences with the mutation and using that alone. Still doesn't call the mutation displayed above. Is it genbank file related? I.e. it isn't genic, and it isn't intragenic because there is no other gene after it in the gb file?

**Comment by benjamincjackson** [comment]
Hallo Matt,

Sorry about this, a stupid bug on my part. Yes, it was because that SNP was after the last gene, but I didn't intend for 3' UTR to be skipped (the 5' UTR was included...).

I've pushed a change to the dev branch that should fix it. You will need to install go, but if you want to use/test a binary that hopefully works you can run `go install github.com/virus-evolution/gofasta@dev` to build one locally (mine goes here: `~/go/bin/gofasta`). Or you can clone the repo, enter it, checkout to the dev branch and issue `go build -o gofasta`.

I will add more tests for the variants routine and include this in the next tagged release.

Ben

**Comment by mattdmem** [comment]
Thanks so much for fixing this quickly. Yes this fixes the bug and I now get the mutation call I was expecting!

**Comment by benjamincjackson** [comment]
Thank you for the helpful issue!

**Comment by benjamincjackson** [comment]
fixed in [version 1.0.0](https://github.com/virus-evolution/gofasta/releases/tag/v1.0.0)
